src/InlineInput/NumberInlineInput.tsx

```diff
@@ -77,21 +80,32 @@ const changeValueStepwise = (
     return nextValue != null ? nextValue : current;
 };
 
-const NumberInlineInput: FC<Props> = ({
+export default ({
     disabled,
     value,
     range,
+    className,
     onChange,
     onChangeComplete = () => {},
-}) => {
+    textAlignLeft,
+    valueIsValidCallback,
+}: NumberInlineInput) => {
     useValidatedRangeOrValues(range);
 
     return (
         <InlineInput
-            className="number-inline-input"
+            className={`${className} number-inline-input`}
             disabled={disabled}
             value={String(value)}
-            isValid={newValue => isValid(Number(newValue), range)}
+            isValid={newValue => {
+                const validity = isValid(Number(newValue), range);
+                if (valueIsValidCallback != null) {
+                    // Then propagate the validity back to parent
+                    valueIsValidCallback(validity);
+                    return true;
```

Comment:
return valueIsValidCallback(validity);

Reply:
Also keep in mind  what we discussed i.e.

onValidationChanged?: (value: boolean) => void;
autoStyleInvalid: boolean;